kong/plugins/mashape-analytics/schema.lua

```diff
@@ -4,6 +4,9 @@ return {
     environment = { type = "string" },
     batch_size = { type = "number", default = 100 },
     log_body = { type = "boolean", default = false },
-    delay = { type = "number", default = 2 }
+    delay = { type = "number", default = 2 },
+    host = { required = true, type = "string", default = "socket.analytics.mashape.com" },
+    port = { required = true, type = "number", default = 80 },
+    host = { required = true, type = "string", default = "/1.0.0/batch" }
```

Comment:
Excuse my ignorance, but is the :fire: icon supposed to highlight the bug in the line above it?